### PR TITLE
fix(performRequest): handle html http responses

### PR DESCRIPTION
**Pull request by corthmann**

### CODE REVIEW

File: bigcommerce/orders_test.go [hunk 1]
```diff
@@ -270,6 +270,73 @@ func TestOrderService_NewWithError(t *testing.T) {
 	assert.EqualError(t, err, BadRequestErrorMessage)
 }
 
+func TestOrderService_NewWithErrorContentHTML(t *testing.T) {
```
Comment (Davejkane) [inline bigcommerce/orders_test.go:273]: 😱 underscore !

**Review by Davejkane — APPROVED** [review]
LGTM (apart from that underscore. That hurt.)

File: bigcommerce/orders_test.go (hunk 1 above)
Comment (corthmann) [reply bigcommerce/orders_test.go:273]: @Davejkane this underscore is intentional and follows the naming convention described in the golang testing package: https://golang.org/pkg/testing/#hdr-Examples

Comment (Davejkane) [reply bigcommerce/orders_test.go:273]: Touché

**Review by Davejkane — APPROVED** [review]

**Pull request merged**